tests/fixtures.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+from labware_domain_models import BarcodedSbsLabware
+from labware_domain_models import LabwareDefinition
+
+GENERIC_UUID = uuid.uuid4()
+
+GENERIC_LABWARE_DEFINITION_KWARGS = {
+    "name": "12-well",
+    "uuid": GENERIC_UUID,
+    "row_count": 3,
+    "column_count": 4,
+    "center_of_a1": (8, 7),
+    "plate_height": 13.4,
+    "distance_to_liquid": 4,
+    "row_center_to_center_spacing": 12,
+    "column_center_to_center_spacing": 15,
+}
+
+GENERIC_LABWARE_DEFINITION = LabwareDefinition(
+    **GENERIC_LABWARE_DEFINITION_KWARGS  # type: ignore
```

Comment:
could probably do Dict[str, Any] at least as a type instead of completely ignoring